### PR TITLE
Helm standards

### PR DESCRIPTION
**Pull request by bivas**

### CODE REVIEW

File: stable/presto/templates/_helpers.tpl [hunk 1]
```diff
@@ -30,7 +30,7 @@ If release name contains chart name it will be used as a full name.
 {{- else -}}
 {{- $name := default .Chart.Name .Values.nameOverride -}}
 {{- if contains $name .Release.Name -}}
-{{- .Release.Name | trunc 63 | trimSuffix "-" -}}
+{{- printf "%s-coordinator" .Release.Name | trunc 63 | trimSuffix "-" -}}
```
Comment (pavius) [inline stable/presto/templates/_helpers.tpl:33]: In other charts we don't trimSuffix, up to you

File: stable/presto/templates/configmap-v3io-config.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
```
Comment (pavius) [inline stable/presto/templates/configmap-v3io-config.yaml]: Should be `v3io-config-configmap.yaml` as per convention

File: stable/presto/templates/configmap-v3io-config.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+  v3io-presto.sh: |
+    #!/usr/bin/env bash
+    set -e
+
+{{ include "v3io-configs.script.lookupService" . | indent 4 }}
+
+    /bin/bash -x {{ .Values.server.config.mapPath }}/docker-presto.sh
+
+---
```
Comment (pavius) [inline stable/presto/templates/configmap-v3io-config.yaml]: I think we should not have separators in the templates

File: stable/spark/templates/_helpers.tpl [hunk 4]
```diff
@@ -4,23 +4,47 @@
 Create fully qualified names.
 */}}
 
-{{- define "master-name" -}}
+{{- define "spark.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
```
Comment (pavius) [inline stable/spark/templates/_helpers.tpl]: Can't we simplify to `{{- printf "%s-spark" .Release.Name | trunc 63 -}}`?

File: stable/spark/templates/configmap-v3io-config.yaml [hunk 5]
```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
```
Comment (pavius) [inline stable/spark/templates/configmap-v3io-config.yaml]: v3io-config-configmap

File: stable/presto/templates/configmap-v3io-config.yaml (hunk 3 above)
Comment (bivas) [reply stable/presto/templates/configmap-v3io-config.yaml]: You're correct. I think it's a testing relic

File: stable/spark/templates/_helpers.tpl (hunk 4 above)
Comment (bivas) [reply stable/spark/templates/_helpers.tpl]: This it the Helm Chart convention. We name charts by their name and (usually) without suffix

File: stable/presto/templates/_helpers.tpl (hunk 1 above)
Comment (bivas) [reply stable/presto/templates/_helpers.tpl:33]: Need to align the rest with trimming of suffixes

**Pull request merged**